static/inline/js/inline.js

```diff
@@ -360,7 +363,8 @@
                             breakpoint: 769,
                             settings: {
                                 arrows: false,
-                                centerPadding: '0px',
+                                centerPadding: Utils.isset(options, 'videos_carousel_center_padding') ? options.videos_carousel_center_padding : '35% 0 0',
```

Comment:
This should read as:

`centerPadding: options.videos_carousel_center_padding`

`videos_carousel_center_padding` should exists in `defaults/` and it should have a default value

Reply:
updating